dev/io.openliberty.jakartaee9.internal_fat/fat/src/io/openliberty/jakartaee9/internal/tests/EE9FeatureCompatibilityTest.java

```diff
@@ -118,6 +118,9 @@ private static Set<String> getCompatibleFeatures(Set<String> allFeatures, boolea
         compatibleFeatures.removeAll(FeatureUtilities.allMpFeatures());
         compatibleFeatures.addAll(FeatureUtilities.compatibleMpFeatures(EEVersion.EE9));
 
+        //MpTelemetry-1.1 is compatible with EE9 but not in Mp50
+        compatibleFeatures.add("mpTelemetry-1.1");
```

Comment:
Should `mpTelemetry-1.1` have been added to the set returned by `FeatureUtilities.compatibleMpFeatures` on the line above?

Reply:
Maybe? At the moment, `compatibleMpFeatures()` finds the compatible features by reading the feature sets from `MicroProfileActions`. It doesn't find `mpTelemetry-1.1` because it's not in the MP 5.0 set. We could actually add it there I guess, since MP 5.0 doesn't contain any other mpTelemetry features.

Reply:
There is a standalone list for different MP levels, and mpTelemetry-1.1 probably should be added to the standalone list for pre MP 6.1 levels?